src/Microsoft.DotNet.Wpf/src/Themes/XAML/Button.xaml

```diff
@@ -59,632 +59,648 @@ See the LICENSE file in the project root for more information.
-                    </theme:ButtonChrome>
-                    <ControlTemplate.Triggers>
-                        <Trigger Property="IsKeyboardFocused"
-                                 Value="true">
-                            <Setter TargetName="Chrome"
-                                    Property="RenderDefaulted"
-                                    Value="true"/>
-                        </Trigger>
-                        <Trigger Property="ToggleButton.IsChecked"
-                                 Value="true">
-                            <Setter TargetName="Chrome"
-                                    Property="RenderPressed"
-                                    Value="true"/>
-                        </Trigger>
-                        <Trigger Property="IsEnabled"
-                                 Value="false">
-                            <Setter Property="Foreground"
-                                    Value="{DynamicResource {x:Static SystemColors.GrayTextBrushKey}}"/>
-                        </Trigger>
-                    </ControlTemplate.Triggers>
-                </ControlTemplate>
-            </Setter.Value>
-        </Setter>
-    </Style>
+<LinearGradientBrush x:Key="ButtonNormalBackgroundFill"
+                     EndPoint="0.5,1"
+                     StartPoint="0.5,0">
+    <LinearGradientBrush.GradientStops>
+        <GradientStop Color="#FFFFFFFF"
+                      Offset="0"/>
+        <GradientStop Color="#FFE3EBF3"
+                      Offset="0.5"/>
+        <GradientStop Color="#FFD0DCEB"
+                      Offset="0.5"/>
+        <GradientStop Color="#FFA6B8CF"
+                      Offset="1"/>
+    </LinearGradientBrush.GradientStops>
+</LinearGradientBrush>
+
+<SolidColorBrush x:Key="ButtonBorder"
+                 Color="#FF2B4F82"/>
+
+<Style x:Key="BaseButtonStyle"
+       TargetType="{x:Type ButtonBase}">
+    <Setter Property="FocusVisualStyle"
+            Value="{StaticResource ButtonFocusVisual}"/>
+    <Setter Property="Background"
+            Value="{StaticResource ButtonNormalBackgroundFill}"/>
+    <Setter Property="BorderBrush"
+            Value="{StaticResource ButtonBorder}"/>
+    <Setter Property="Foreground"
+            Value="{DynamicResource {x:Static SystemColors.ControlTextBrushKey}}"/>
+    <Setter Property="HorizontalContentAlignment"
+            Value="Center"/>
+    <Setter Property="VerticalContentAlignment"
+            Value="Center"/>
+    <Setter Property="Template">
+        <Setter.Value>
+            <ControlTemplate TargetType="{x:Type ButtonBase}">
+                <theme:ButtonChrome x:Name="Chrome"
+                                    BorderBrush="{TemplateBinding BorderBrush}"
+                                    Fill="{TemplateBinding Background}"
+                                    RenderDefaulted="{TemplateBinding Button.IsDefaulted}"
+                                    RenderMouseOver="{TemplateBinding IsMouseOver}"
+        
```

Comment:
Explicitly add IsKeyboardFocused false check here, otherwise you're depending on evaluation order of the triggers to get the correct BorderBrush.